test/info.py

```diff
@@ -62,6 +62,14 @@
     'lpc11u35_archlink_if': 0x9013,
 }
 
+# Maps a target application filename (sans extension) to the ID of the board 
+# it is built to run on. This mapping is also used to display a board name 
+# in the test output. Thus, name the target applications after the  board to
+# avoid confusing results.
+#
+# One-to-many is fine. Many-to-one is fine as long as only one (or none) target 
+# application is found on disk per connected board ID
+#
```

Comment:
I'll update the comment now...

On Wed, Apr 13, 2016 at 10:56 AM, Russ Butler <notifications@github.com>
wrote:

> In test/info.py
> <https://github.com/mbedmicro/DAPLink/pull/89#discussion_r59574510>:
>
> > @@ -62,6 +62,14 @@
> >      'lpc11u35_archlink_if': 0x9013,
> >  }
> >
> > +# Maps a target application filename (sans extension) to the ID of the board
> > +# it is built to run on. This mapping is also used to display a board name
> > +# in the test output. Thus, name the target applications after the  board to
> > +# avoid confusing results.
> > +#
> > +# One-to-many is fine. Many-to-one is fine as long as only one (or none) target
> > +# application is found on disk per connected board ID
> > +#
>
> If it isn't easy to update the comment then I'll take care of it in
> another PR.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/mbedmicro/DAPLink/pull/89/files/60d34809b9a349aaca256f99756d8f475313fede#r59574510>
>